src/main/cpp/sim/run_stride.cpp

```diff
@@ -132,9 +147,17 @@ void run_stride(bool track_index_case,
 	// Run the simulation.
 	const unsigned int num_days = pt_config.get<unsigned int>("run.num_days");
 	vector<unsigned int> cases(num_days);
+	Stopwatch<> run_clock("run_clock");
+
 	for (unsigned int i = start_day; i < start_day + num_days; i++) {
 		cout << "Simulating day: " << setw(5) << i;
-		coord.timeStep();
```

Comment:
zal even kijken en de run_stride terug wat 'in line' met die van de master zetten